schemas/qwc-map-viewer.json

```diff
@@ -117,6 +117,13 @@
           "items": {
             "type": "string"
           }
+        },
+        "public_paths": {
+          "description": "List of paths that will be accessible without a JWT identity",
```

Comment:
"Optional: List of paths that will be accessible without a JWT identity, if `AUTH_REQUIRED` is set."